api/src/main/java/org/openmrs/util/OpenmrsClassLoader.java

```diff
@@ -590,7 +590,7 @@ protected static void nullInstance(Object instance) {
 							    field.getName(), instance.getClass().getName(), valueClass.getName());
 						} else {
 							field.set(instance, null);
-							log.debug("Set field {} to null in class", field.getName(), instance.getClass().getName());
+							log.debug("Set field {} to null in class {}", field.getName(), instance.getClass().getName());
```

Comment:
Was the comma before or after the field name?

Reply:
The comma was after field.getName()